scripts/konnect-funcs/supportedtypes.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+var supportedTypes = []supportedTypesT{
+	{
+		Package: "v1",
+		Types: []templateDataT{
+			{
+				Type:              "KongConsumer",
+				KonnectStatusType: "konnectv1alpha1.KonnectEntityStatusWithControlPlaneRef",
+				ReceiverName:      "c",
```

Comment:
This might be useful if people define additional methods on those objects. Those other `func`s would have to define `o` as receiver as well otherwise I believe there a linter (not sure which one) which complains about inconsistent receiver names.

Reply:
https://github.com/Kong/kubernetes-configuration/blob/50f12a24fd51ef2fea501b09397b49f532c3555d/.golangci.yaml#L122 excludes the `api/` directory so linter complaints won't happen. Anyway, being forced to use `o` receiver is not that bad, in the end, it's always an object. 😃 Won't fight over it though, it's not that important.